src/app/project/[id]/edit/page.tsx

```
@@ -84,35 +26,76 @@ export default function ProductEdit() {
+          setIsLoading(false)
+          return
+        }
+
+        // API 호출로 프로젝트 데이터 가져오기
+        const response = await apiCall<any>(`/api/project/${id}`, "GET")
 
-        // 목 데이터 사용
-        const data = mockProductData
+        if (response.error) {
+          console.error("프로젝트 데이터 로드 실패:", response.error)
+          setIsLoading(false)
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)